tools/stern/src/Stern/API.hs

```diff
@@ -402,15 +402,15 @@ getUserData uid = do
   convs <- Intra.getUserConversations uid
   clts <- Intra.getUserClients uid
   notfs <- Intra.getUserNotifications uid
-  consent <- Intra.getUserConsentValue uid
-  consentLog <- Intra.getUserConsentLog uid
+  consent <- (Intra.getUserConsentValue uid <&> Just) `catchE` \_ -> pure Nothing
```

Comment:
do these catch internal errors as well?  do we want that?

Reply:
I don't know, I just noticed that these fail and make the whole request fail, which is unnecessary, IMO.

Reply:
made a change, see comment below